lib/raven/configuration.rb

```diff
@@ -261,6 +262,13 @@ def verify!
       end
     end
 
+    def verify_messages
```

Comment:
I agree - but it's a bit hard to check ;) `capture_allowed_by_callback?` is always returning `true` if the message is `nil`, so we'd need to send a bogus value.

I guess really interesting is only a filter that will always block, like `Raven.configuration.should_capture = -> { false }`.